SampleComponentLibraryApp/SampleComponentLibraryApp/SceneDelegate.swift

```diff
@@ -18,5 +19,10 @@ class SceneDelegate: UIResponder, UIWindowSceneDelegate {
         window = UIWindow(windowScene: windowScene)
         window?.rootViewController = navigationController
         window?.makeKeyAndVisible()
+
+        guard let window else { return }
```

Comment:
Super-minor suggestion: if we store the UIWindow() result first in a local non-optional it avoids the need for the guard (and the optional `?` before calling functions).

```swift
let newWindow = UIWindow(windowScene: windowScene)
window = newWindow
newWindow.rootViewController = navigationController
// … etc
```

Reply:
thanks for the suggestion, updated!